src/modules/rppi_validate.hpp

```diff
@@ -12,6 +12,93 @@
 #include <hip/hip_runtime_api.h>
 #endif
 
+inline RppLayoutParams get_layout_params(RpptLayout layout, Rpp32u channels)
+{
+    RppLayoutParams layoutParams;
+    if(layout == RpptLayout::NCHW)
+    {
+        if (channels == 1) // PLN1
+        {
+            layoutParams.channelParam = 1;
+            layoutParams.bufferMultiplier = 1;
+        }
+        else if (channels == 3) // PLN3
+        {
+            layoutParams.channelParam = 3;
+            layoutParams.bufferMultiplier = 1;
+        }
+    }
+    else if(layout == RpptLayout::NHWC)
+    {
+        if (channels == 3) // PKD3
+        {
+            layoutParams.channelParam = 1;
+            layoutParams.bufferMultiplier = 3;
+        }
+    }
+
+    return layoutParams;
+}
+
+// inline void copy_roiTensor(RpptROIPtr roiTensorPtrSrc, rpp::Handle& handle)
```

Comment:
remove commented out codes if it is not needed.

Reply:
Done. Cleaned up this rppi_validate file for just some formatting changes, tabs to spaces, and removal of unwanted commented code.